p2/index.qmd

```diff
@@ -32,27 +33,28 @@ Install the packages by typing in the following commands into RStudio (you do no
 
 If you need to install any of these packages use:
 
-::: {.panel-tabset group="language"}
+::: {.panel-tabset}
 
 ## R
 
-```{r}
+```{r} 
 #| eval: false
 if (!require("pak")) install.packages("pak")
 pak::pkg_install(c("sf", "tidyverse", "remotes"))
 # GitHub pkgs
 pak::pkg_install("Nowosad/spDataLarge")
 ```
 
-```{r}
+``` {r}
 library(sf)          # vector data package 
 library(tidyverse)   # tidyverse packages
 library(spData)  # spatial data package 
 ```
 
 ## Python
 
-```{python, eval=FALSE}
+``` {python}
```

Comment:
👍

Reply:
But without the space I think is better:

```suggestion
```{python}
```

Reply:
Thanks for pointing these out. I will fix them in the future— the added space was by mistake and not intentional.